content/en/llm_observability/setup/auto_instrumentation.md

```diff
@@ -4,18 +4,24 @@ aliases:
     - /tracing/llm_observability/auto_instrumentation
     - /llm_observability/auto_instrumentation
 further_reading:
-    - link: '/llm_observability/setup/sdk'
+    - link: '/llm_observability/setup/sdk/python'
       tag: 'Documentation'
       text: 'Learn about the LLM Observability SDK for Python'
+    - link: '/llm_observability/setup/sdk/nodejs'
+      tag: 'Documentation'
+      text: 'Learn about the LLM Observability SDK for Node.js'
 ---
 
 <div class="alert alert-info">Datadog offers a variety of artificial intelligence (AI) and machine learning (ML) capabilities. The <a href="/integrations/#cat-aiml">AI/ML integrations on the Integrations page and the Datadog Marketplace</a> are platform-wide Datadog functionalities. <br><br> For example, APM offers a native integration with OpenAI for monitoring your OpenAI usage, while Infrastructure Monitoring offers an integration with NVIDIA DCGM Exporter for monitoring compute-intensive AI workloads. These integrations are different from the LLM Observability offering.</div>
 
+{{< tabs >}}
```

Comment:
:+1: nice - i like the tab approach. I would propose taking it a step further by just having the code-snippets be language specific and leave the bits in between (like definitions, intros to the product terms) to be shared. 

I can definitely see the duplicated info between the Python and Node.js pages going out of sync real fast.

I also realize this is a big refactor to these docs so I'll leave it up to you @sabrenner

Reply:
We can merge as-is for now, and I can take on that refactor in a later PR!

Reply:
thanks @cswatt, really appreciate that! i'm happy to help out with that when/if needed 😄